lookout/style/typos/general.py

```diff
@@ -0,0 +1,83 @@
+from typing import Any, Mapping, Optional
```

Comment:
"general.py" does not seem to be a good module name. As of now, it looks like "train_dataset_preparation.py"